OrbitGl/CaptureWindow.cpp

```diff
@@ -480,6 +480,25 @@ void CaptureWindow::MouseWheelMoved(int a_X, int a_Y, int a_Delta,
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void CaptureWindow::MouseWheelMovedHorizontally(int a_X, int a_Y, int a_Delta,
+                                                bool a_Ctrl) {
+  if (a_Delta == 0) return;
+
+  // delta < 0 is left.
+  int delta = -a_Delta / abs(a_Delta);
```

Comment:
delta is 1 or -1, should we have something like: int delta = a_Delta < 0 : 1 : -1; ? Or invert it so that we pass "delta" and not "-delta" below.

Reply:
The reason I had written it like that was to keep a parallel with `MouseWheelMoved`, which does the same. I changed to `int delta = a_Delta < 0 ? 1 : -1` also in the `MouseWheelMoved`s. But I would keep passing `-delta` for such parallel.

Reply:
Once I manage to push, that is...

Reply:
Cool, thanks.